src/Core/Localization/CLDR/Reader.php

```diff
@@ -440,20 +463,24 @@ protected function mapLocaleData(SimpleXMLElement $xmlLocaleData, $supplementalD
                     )
                 ) {
                     $aliasNumSys = $matches[1];
-                    $localeData->percentPatterns[$numberSystem] = $localeData->percentPatterns[$aliasNumSys];
+                    $percentPatterns[$numberSystem] = $localeData->getPercentPatterns()[$aliasNumSys];
```

Comment:
It seems you can use `$percentPatterns` here:
```suggestion
                    $percentPatterns[$numberSystem] = $percentPatterns[$aliasNumSys];
```